biomaj/bank.py

```diff
@@ -525,10 +527,36 @@ def save_session(self):
 
         self.bank = self.banks.find_one({'name': self.name})
 
+    def format_release(self, release, reverse=False):
+        """
+        Check the release does not contains any dot '.' character that should make MongoDB complain
+
+        :param release: release number to check
+        :type release: str
+        :param reverse: Do the reverse replacement
+        :type reverse: bool
+        :return: Release string with dot '.' replaced with character set in config (release.replace)
+        :rtype: str
+        """
+        if not self.config.get('release.replacedot'):
```

Comment:
In anyway, we will also have to call this method `format_release` with argument `reverse=True` when displaying info to the user, e.g. when calling `biomaj-cli.py --status`
You agree with that?